roles/tests-integration/tests/common/mod.rs

```diff
@@ -146,7 +146,10 @@ impl TemplateProvider {
     }
 
     fn stop(&self) {
-        let _ = self.bitcoind.client.stop().unwrap();
+        match self.bitcoind.client.stop() {
+            Ok(_) => (),
+            Err(e) => eprintln!("Failed to stop bitcoind: {:?}", e),
```

Comment:
this is a bit too YOLO and doesn't fully fix the root of the problem

we need to figure out how to properly stop `bitcoind` via the crate APIs

please check the updated description of https://github.com/stratum-mining/stratum/issues/1288